### ISSUE TITLE
[Bug]: Cannot running any problem

### ISSUE THREAD

**Issue by falconshark** [Whisky-App/Whisky#1003]
### Description

Today I had changed my system display language from "Chinese (Hong Kong)" to "English (US)", then I could never open any program such as Steam and R2modman now.
It do not provide any error message, so I could not confirm which part made this problem.

(Tried fully delete bottle, but not use)

### Steps to reproduce

1. Change the main system language of Mac OS
2. Start pinned App such as Steam
3. Nothing displayed.

### Expected behaviour

Started program such as Steam, nothing display.
Do not get any error message.

### Logs

```bash
Whisky Version: 2.3.2
Date: 2024-05-30T12:55:43Z
macOS Version: 14.5.0

Bottle Name: Steam
Bottle URL: /Users/sardo/Library/Containers/com.isaacmarovitz.Whisky/Bottles/C03528DB-7E29-4353-8850-EF71048032EA

Wine Version: 7.7.0
Windows Version: win10
Enhanced Sync: msync

Metal HUD: false
Metal Trace: false

Arguments: start /unix /Users/sardo/Library/Containers/com.isaacmarovitz.Whisky/Bottles/C03528DB-7E29-4353-8850-EF71048032EA/drive_c/Program Files (x86)/Windows Media Player/wmplayer.exe

Environment:
{
    "DXVK_ASYNC" = 1;
    "GST_DEBUG" = 1;
    WINEDEBUG = "fixme-all";
    WINEESYNC = 1;
    WINEMSYNC = 1;
    WINEPREFIX = "/Users/sardo/Library/Containers/com.isaacmarovitz.Whisky/Bottles/C03528DB-7E29-4353-8850-EF71048032EA";
}
```

### What version of Whisky are you using?

2.3.2

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by plyght** [comment]
@falconshark If you've only tried to run Steam and it seems to take a while, just be patient as Steam uses Chromium and typically has a longer loading time.

If you've attempted to run other games and they still won't load, I highly recommend reinstalling Whisky. You can save your bottles if you'd like, and I can walk you through that process if needed.

**Comment by falconshark** [comment]
> @falconshark If you've only tried to run Steam and it seems to take a while, just be patient as Steam uses Chromium and typically has a longer loading time.
> 
> If you've attempted to run other games and they still won't load, I highly recommend reinstalling Whisky. You can save your bottles if you'd like, and I can walk you through that process if needed.

Thanks for your reply ! But unlucky, I had tried to reinstall Whisky and run other program without Steam, and it still cannot open...

![截圖 2024-06-04 上午11 09 36](https://github.com/Whisky-App/Whisky/assets/2662715/3084b234-92c0-4cec-9d48-97b335525395)

It's look like the msync had been stucked...

**Comment by plyght** [comment]
> > @falconshark If you've only tried to run Steam and it seems to take a while, just be patient as Steam uses Chromium and typically has a longer loading time.
> > If you've attempted to run other games and they still won't load, I highly recommend reinstalling Whisky. You can save your bottles if you'd like, and I can walk you through that process if needed.
> 
> Thanks for your reply ! But unlucky, I had tried to reinstall Whisky and run other program without Steam, and it still cannot open...
> 
> ![截圖 2024-06-04 上午11 09 36](https://private-user-images.githubusercontent.com/2662715/336280143-3084b234-92c0-4cec-9d48-97b335525395.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MDM4OTksIm5iZiI6MTcxNzUwMzU5OSwicGF0aCI6Ii8yNjYyNzE1LzMzNjI4MDE0My0zMDg0YjIzNC05MmMwLTRjZWMtOWQ0OC05N2IzMzU1MjUzOTUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDRUMTIxOTU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTA1OTY0ZDIzNDIwYmQ3NDEyMzlhMTEwMTdkZmNjZmZkNzA5NTBiYjk1ZGI5M2MyZmZkNGVjYTFhNjFkM2Q5YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.enTnui6B89yTpEjkI27-z87-jvCP0CBh9x_InTokUqk)
> 
> It's look like the msync had been stucked...

@falconshark yeah, try [turning Msync off](https://github.com/Whisky-App/Whisky/assets/81438024/efd39003-073a-4115-8216-cebb7b369593).

**Comment by falconshark** [comment]
> > > @falconshark If you've only tried to run Steam and it seems to take a while, just be patient as Steam uses Chromium and typically has a longer loading time.
> > > If you've attempted to run other games and they still won't load, I highly recommend reinstalling Whisky. You can save your bottles if you'd like, and I can walk you through that process if needed.
> > 
> > 
> > Thanks for your reply ! But unlucky, I had tried to reinstall Whisky and run other program without Steam, and it still cannot open...
> > ![截圖 2024-06-04 上午11 09 36](https://private-user-images.githubusercontent.com/2662715/336280143-3084b234-92c0-4cec-9d48-97b335525395.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1MDM4OTksIm5iZiI6MTcxNzUwMzU5OSwicGF0aCI6Ii8yNjYyNzE1LzMzNjI4MDE0My0zMDg0YjIzNC05MmMwLTRjZWMtOWQ0OC05N2IzMzU1MjUzOTUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDRUMTIxOTU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NTA1OTY0ZDIzNDIwYmQ3NDEyMzlhMTEwMTdkZmNjZmZkNzA5NTBiYjk1ZGI5M2MyZmZkNGVjYTFhNjFkM2Q5YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.enTnui6B89yTpEjkI27-z87-jvCP0CBh9x_InTokUqk)
> > It's look like the msync had been stucked...
> 
> @falconshark yeah, try [turning Msync off](https://github.com/Whisky-App/Whisky/assets/81438024/efd39003-073a-4115-8216-cebb7b369593).

Thanks you very much !
I try to change Msync to Esync, other program is able to running now. However, Steam is still only able to showing the icon on dock, and not thing happen after waiting 10 mins ;)
